### PR TITLE
i18n: Markup Explorer Graph for translation

### LINKED ISSUE #74311: i18n: Mark up Graph for translation
### Requirements ###

Mark up https://github.com/grafana/grafana/tree/v10.1.0/public/app/features/explore/Graph components for translations.

### Background

Please follow #70957 for more info and guidelines.

### PR DESCRIPTION
**Pull request by krishnemd**
**What is this feature?**

i18n integration in Graph component

**Why do we need this feature?**

For integrating i18n in Explorer

**Who is this feature for?**

Explorer users

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana/issues/74311

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/features/explore/Graph/GraphContainer.tsx
Comment (JoaoSilvaGrafana) [inline public/app/features/explore/Graph/GraphContainer.tsx:86]: I think because `MAX_NUMBER_OF_TIME_SERIES` will never be 1, both this and the other plural translations don't actually need to be plural translations, but maybe someone from @grafana/explore-squad can give their opinion on that?

Comment (chauchausoup) [reply public/app/features/explore/Graph/GraphContainer.tsx:86]: Awesome. Should I wait for the squad's message or make changes already @JoaoSilvaGrafana  ?

Comment (JoaoSilvaGrafana) [reply public/app/features/explore/Graph/GraphContainer.tsx:86]: Feel free to make the changes now, but I'd wait for the squad for approval afterwards 👍

Comment (ifrost) [reply public/app/features/explore/Graph/GraphContainer.tsx:86]: Yep, that's a good point. It should never be 1 so it doesn't need to be a plural 👍

**Comment by chauchausoup** [comment]
@JoaoSilvaGrafana I have made the changes.

**Review by JoaoSilvaGrafana — APPROVED** [review]
Lgtm!

**Pull request merged**
